src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetChangelogPage.tsx

```diff
@@ -74,49 +78,55 @@ export default function ReleaseApiDataSetChangelogPage() {
       >
         Back to API data set details
       </Link>
+
       <LoadingSpinner loading={isLoadingDataSet || isLoadingChanges}>
-        <div className="govuk-grid-row">
-          <div className="govuk-grid-column-three-quarters">
-            <span className="govuk-caption-l">API data set changelog</span>
-            <h2>{dataSet?.title}</h2>
-          </div>
-        </div>
+        {!errorFetchingDataSet && !errorFetchingChanges ? (
```

Comment:
As the requests are independent, there's no need to couple both into the same conditional like this. 

We could just check that only `dataSet` is defined (see earlier comments about not need to use `isError`) and render as much UI as possible before showing the error message for just the section relating to the changelog specifically.

Specifically would be looking for the following:

```tsx
{dataSet ? (
  ...
) : (
  <WarningMessage>Could not load API data set</WarningMessage>
)
```

Reply:
Done ta